src/pymor/models/iosys.py

```diff
@@ -700,6 +701,132 @@ def __mul__(self, other):
                         T=self.T, initial_data=initial_data, time_stepper=time_stepper, num_values=self.num_values,
                         solver_options=self.solver_options)
 
+    def impulse_resp(self, mu=None, return_solution=False):
+        """Compute impulse response from all inputs.
+
+        Parameters
+        ----------
+        mu
+            |Parameter values| for which to solve.
+        return_solution
+            If `True`, the model solution for the given |parameter values| `mu` is returned.
+
+        Returns
+        -------
+        y
+            Impulse response as a 3D |NumPy array| where
+            `y.shape[0]` is the number of snapshots,
+            `y.shape[1]` is the number of outputs, and
+            `y.shape[2]` is the number of inputs.
```

Comment:
This feels like matlab style. I'd expect first inputs, then outputs, then time.

Reply:
I think this way is better as it is similar to what `freq_resp` returns.